tests/modules/graphql_resolver_builder_test/graphql_resolver_builder_test.info.yml

```diff
@@ -0,0 +1,9 @@
+type: module
+name: GraphQL Resolver Builder Test
+description: Tests the graphql resolver builder functionality.
+package: Testing
+core_version_requirement: ^8 || ^9
+hidden: TRUE
+dependencies:
+  - graphql:graphql
+  - drupal:typed_data
```

Comment:
this line can be removed, graphql module already depends on typed_data module.

Or actually we don't need any dependencies at all, the module only uses Drupal core.